plugins/nevermind.py

```diff
@@ -0,0 +1,14 @@
+from errbot import BotPlugin, re_botcmd
+
+class Nevermind(BotPlugin):
+    """
+    Doesn't mind
+    """
+
+    @staticmethod
+    def nm_msg():
+        return "I'm sorry :("    
```

Comment:
I created an issue, please see https://github.com/coala/corobo/issues/23#issuecomment-303944269 

It is in discuss mode, please leave a feedback there :D